### PR TITLE
Changes to RT-1.5: BGP Prefix Limit test

### PR DESCRIPTION
**Pull request by arvbaska1**
Made below changes to the the BGP RT-1.5 test in verifyBGPTelemetry
- Changed order to verify first the Received routes leaf, and then the Installed routes.
- Changed await timeout to 2mins for subscription for Received routes.

### CODE REVIEW

**Review by guoshiuan — APPROVED** [review]
b/274669361 has the internal test results.

File: feature/bgp/prefixlimit/ate_tests/bgp_prefix_limit_test/bgp_prefix_limit_test.go [hunk 1]
```diff
@@ -384,12 +384,12 @@ func (tc *testCase) verifyBGPTelemetry(t *testing.T, dut *ondatra.DUTDevice) {
...
 	statePath := gnmi.OC().NetworkInstance(*deviations.DefaultNetworkInstance).Protocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, "BGP").Bgp()
 	prefixes := statePath.Neighbor(ateDst.IPv4).AfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV4_UNICAST).Prefixes()
-	if got, ok := gnmi.Watch(t, dut, prefixes.Installed().State(), time.Minute, compare).Await(t); !ok {
-		t.Errorf("Installed prefixes v4 mismatch: got %v, want %v", got, installedRoutes)
-	}
-	if got, ok := gnmi.Watch(t, dut, prefixes.Received().State(), time.Minute, compare).Await(t); !ok {
+	if got, ok := gnmi.Watch(t, dut, prefixes.Received().State(), 2*time.Minute, compare).Await(t); !ok {
 		t.Errorf("Received prefixes v4 mismatch: got %v, want %v", got, installedRoutes)
 	}
+	if got, ok := gnmi.Watch(t, dut, prefixes.Installed().State(), 2*time.Minute, compare).Await(t); !ok {
```
Comment (Ankur19) [inline feature/bgp/prefixlimit/ate_tests/bgp_prefix_limit_test/bgp_prefix_limit_test.go:390]: Do we also need to wait 2 mins for the `v6` routes?

**Review by Ankur19 — APPROVED** [review]
LGTM!

Optional: For consistency we should also wait 2 mins for v6 routes?

**Review by prinikasn — COMMENTED** [review]
/gcbrun

File: feature/bgp/prefixlimit/ate_tests/bgp_prefix_limit_test/bgp_prefix_limit_test.go (hunk 1 above)
Comment (guoshiuan) [reply feature/bgp/prefixlimit/ate_tests/bgp_prefix_limit_test/bgp_prefix_limit_test.go:390]: Had offline discussion with Ankur: while the test is waiting for v4 routes, the v6 route subscription is reached; that's probably we never see this issue for v6 routes.

**Pull request merged**